autoload/lsp/lsp.vim

```diff
@@ -856,7 +861,12 @@ enddef
 # location
 export def Hover(cmdmods: string)
   var lspserver: dict<any> = buf.CurbufGetServerChecked('hover')
-  if lspserver->empty()
+  if lspserver->empty() && &keywordprg !=# ':LspHover'
+    try
+      execute 'normal! K'
```

Comment:
What if people don't use K for looking up keywords? I mean this is a rare case, but possible

Reply:
> Okay, just tried this out. It can actually get a bit annoying in C, when you try to inspect some words and land in the manpages.

True, I forgot about the default value of `&keywordprg`. Added a check

Reply:
Much appreciated